homeassistant/components/mobile_app/webhook.py

```diff
@@ -125,8 +125,11 @@ async def validate_and_run(hass, config_entry, data):
                 data = schema(data)
             except vol.Invalid as ex:
                 err = vol.humanize.humanize_error(data, ex)
-                _LOGGER.error("Received invalid webhook payload: %s", err)
-                return empty_okay_response()
+                if "state" in err:
+                    _LOGGER.debug("Webhook data incomplete, continuing: %s", err)
```

Comment:
I don't think we should do this. We're introducing an inconsistency in our data model so now any downstream data consumer can now no longer trust the data.